app/src/main/java/com/nutomic/syncthingandroid/service/RunConditionMonitor.java

```diff
@@ -58,6 +60,7 @@ public void onStatusChanged(int which) {
     private final Context mContext;
     @Inject SharedPreferences mPreferences;
     private ReceiverManager mReceiverManager;
+    private String mRunDecisionExplanation = "";
```

Comment:
It would be cleaner if you just store a pointer to the string resource here, and only do `getString()` once you display it to the user. Use `@StringRes int mRunDecisionExplanation` for proper typing.